README.md

```diff
@@ -162,6 +162,8 @@ require('telescope').setup{
     file_previewer = require'telescope.previewers'.cat.new, -- For buffer previewer use `require'telescope.previewers'.vim_buffer_cat.new`
     grep_previewer = require'telescope.previewers'.vimgrep.new, -- For buffer previewer use `require'telescope.previewers'.vim_buffer_vimgrep.new`
     qflist_previewer = require'telescope.previewers'.qflist.new, -- For buffer previewer use `require'telescope.previewers'.vim_buffer_qflist.new`
+    buffer_previewer_wrap = false,
```

Comment:
perhaps instead of this, we can add an autocommand that we fire when we're done loading a previewer.

People can do something like `autocmd User TelescopePreviewerLoaded setlocal nowrap` or similar. I otherwise see maybe having a lot of these in the future. (I think that would work and it only would run in the buffer)